File: server/src/main/kotlin/org/bitlap/server/core/BitlapSessionFactory.kt
package org.bitlap.server.core

import org.bitlap.net.session.SessionFactory

/**
 * Mail: chk19940609@gmail.com
 * Created by IceMimosa
 * Date: 2021/9/4
 */
class BitlapSessionFactory : SessionFactory {

    override fun create(username: String, password: String, sessionConf: Map<String, String>, sessionManager: SessionManager): Session {
        return BitlapSession(username, password, sessionConf, sessionManager)
    }
}

File: server/src/main/scala/org/bitlap/server/core/BitlapSessionFactory.scala
package org.bitlap.server.core

import org.bitlap.net.session.SessionFactory
import org.bitlap.net.session.{ Session, SessionManager }

/**
 *
 * @author 梦境迷离
 * @version 1.0,2021/12/3
 */
class BitlapSessionFactory extends SessionFactory{

  override def create(username: String, password: String, sessionConf: Map[String, String], sessionManager: SessionManager): Session = {
    new BitlapSession(username, password, sessionConf, sessionManager)
  }
}
